### DISCUSSION TITLE
Particle emitter

### DISCUSSION THREAD

**Discussion by Myziak** [enable3d/enable3d#215] [Ideas]
Hello,

What about particle emitter in enable3d?
You could implement elements of a particle system based on solutions such as 'three-nebula' etc.

**Comment by yandeu** [comment]
I have never used `three-nebula`.

Why including it into enable3d? Enable3d is based on three.js so `three-nebula` should just work.

Have you tried it?

**Comment by Myziak** [comment]
This is just an example, that's why I used "etc."
Rather, the point is that enable3d does not have its own particle system - which in 3D scene creation is rather fundamental.
